boring-registry.yaml

```diff
@@ -23,6 +23,11 @@ pipeline:
       expected-commit: 083aa721311a3745ee0e5bd3b68bfce7ed2c1f4c
       tag: v${{package.version}}
 
+  - uses: go/bump
+    with:
+      # CVE-2023-39325 and CVE-2023-3978
+      deps: golang.org/x/net@v0.17.0 google.golang.org/grpc@v1.56.3
```

Comment:
same question re: diffs

Reply:
The grpc one flagged at CI time so I manually added it. We need advisories for these.